plugins/imjournal/imjournal.c

```diff
@@ -1074,6 +1083,14 @@ CODESTARTsetModCnf
 		}
 	}
 
+	/* File create mode is not set */
+	if (cs.fCreateMode == -1) {
+		const int fCreateMode = 0644;
+		LogMsg(0, RS_RET_OK_WARN, LOG_WARNING, "imjournal: filecreatemode is not set, "
+			"using default %04o", fCreateMode);
+		cs.fCreateMode = fCreateMode;
```

Comment:
If my analysis in https://github.com/rsyslog/rsyslog/issues/5375#issuecomment-2798063780 is correct, `cs.fCreateMode` will remain at -1 if `$ModLoad imjournal` is used, because this configuration function will not be called at all. 

Just changing the config line from `module(load="imjournal")` causes rsyslogd to generate a warning.